server/services/FiltersService/index.js

```diff
@@ -66,149 +66,40 @@ class FiltersService {
     }
   }
 
-  barChartMongoQueries = () => {
-    const filters = this.filters
-    const $facet = {}
-    const chrCritFilters = filters.chrcrit_part
-      .filter((f) => f.value === TRUE_STRING)
-      .map((filter) => FILTER_TO_MONGO_VALUE_MAP[filter.name])
-    const includedExcludedFilters = filters.included_excluded
-      .filter((f) => f.value === TRUE_STRING)
-      .map((filter) => FILTER_TO_MONGO_VALUE_MAP[filter.name])
-    const sexAtBirthFilters = filters.sex_at_birth
-      .filter((f) => f.value === TRUE_STRING)
-      .map((filter) => FILTER_TO_MONGO_VALUE_MAP[filter.name])
-
-    const isIncOrExcOnly =
-      !chrCritFilters.length && includedExcludedFilters.length
-    const isCritOnly = chrCritFilters.length && !includedExcludedFilters.length
-    const isIncAndCrit = chrCritFilters.length && includedExcludedFilters.length
-
-    if (isIncAndCrit) {
-      $facet.formInclusion = [
-        {
-          $match: {
-            assessment: INCLUSION_EXCLUSION_CRITERIA_FORM,
+  get filterQueries() {
+    const filterQueries = []
+    for (let filterKey of Object.keys(this.filters)) {
+      const filter = this.filters[filterKey]
+      const includedFilters = filter.filter(({_, value}) => value === TRUE_STRING)
+      const includedValues = includedFilters.map(({name, _}) => FILTER_TO_MONGO_VALUE_MAP[name])
```

Comment:
Can we rename this variable..?
```suggestion
      const includedValues = includedFilters.map(({name, _}) => FILTER_TO_MONGO_VALUE_MAP[name])
```